src/views/patterns/social-sharing.php

```diff
@@ -0,0 +1,11 @@
+<?php declare(strict_types=1);
```

Comment:
This won't work without BE proper registration

Reply:
Do we actually need this?

Reply:
Nope, that's old. Good catch.